README.MD

```diff
@@ -1,33 +1,81 @@
-# This is the official TeaSpeak docker reposetory
-## Build the debian docker
-### Debian
-Just build the docker by executing:
-```
-docker build --tag teaspeak:debian server/debian/
-```
-Running the docker:
-```
-docker run --rm -i teaspeak:debian
-```
-Run with parameters:
-```
-docker run --rm -i teaspeak:debian -Pgeneral.database.url=sqlite://database/TeaData.sqlite
-``` 
+# This is the official TeaSpeak docker repository
+This repository holds all Dockerfiles and bash scripts to build the TeaSpeak Server Images and Web-Client Image
 
-### Ubuntu
-Just build the docker by executing:  
+## Server images
+Basic instructions to build and run the TeaSpeak server images yourself.
+
+### Alpine (~183MB, 13 Layers)
+```shell script
+# Build the Image with the server version 1.4.12 (default)
+docker build -f server/alpine.Dockerfile -t teaspeak/server:1.4.12-alpine .
+
+# alternatively with server version x.y.z
+# docker build -f server/alpine.Dockerfile --build-arg TEASPEAK_VERSION=x.y.z -t teaspeak/server:x.y.z-alpine .
+
+# run the Alpine based server image
+docker run -d \
+  -v teaspeak_logs:/ts/logs \
+  -v teaspeak_db:/ts/database \
+  -v teaspeak_files:/ts/files \
+  -v teaspeak_certs:/ts/certs \
+  -p 9987:9987/udp -p 10101:10101/tcp -p 30303:30303 \
```

Comment:
The web client requires port 9987 TCP.

Reply:
Thanks for the heads up! Will change that ASAP